contrib/incident-response/infrastructure/modules/turbinia/main.tf

```diff
@@ -0,0 +1,259 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+# Random ID for creating unique resource names.
+# TODO: Allow overide by config
+resource "random_id" "infrastructure-random-id" {
+  byte_length = 8
+}
+
+# Enable GCP services individually.
+resource "google_project_service" "cloudfunctions" {
```

Comment:
Is there a specific reason to enable the services individually (besides dependency for the resources created below)? It might be cleaner to add all the services into a list and then use the count of that list to enable the services within one block, like:

```
locals = {
  services_list = [
    "admin.googleapis.com",
    "appengine.googleapis.com",]
} 

resource "google_project_service" "main" {
  count              = length(local.services_list)
  project            = var.project_id
  service            = local.services_list[count.index]
  disable_on_destroy = "false"
}
```

Reply:
That's nice! I didn't know about that pattern. Changed now.